From 258b500a315f6982bc031d3a953f4e3f178b823c Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Thu, 15 Oct 2015 17:52:20 -0400
Subject: [PATCH 0600/1936] EnvlopeFreeEdit: Don't Request Duplicate Info

This should reduce the traffic created by this widget spawning
---
 src/UI/EnvelopeFreeEdit.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/UI/EnvelopeFreeEdit.cpp b/src/UI/EnvelopeFreeEdit.cpp
index c4d5dec3..8b7f2800 100644
--- a/src/UI/EnvelopeFreeEdit.cpp
+++ b/src/UI/EnvelopeFreeEdit.cpp
@@ -24,8 +24,8 @@ void EnvelopeFreeEdit::init(void)
 
     //register for non-bulk types
     for(int i=0; i<MAX_ENVELOPE_POINTS; ++i) {
-        oscRegister((string("Penvdt") + to_s(i)).c_str());
-        oscRegister((string("Penvval") + to_s(i)).c_str());
+        osc->createLink(loc+string("Penvdt") + to_s(i), this);
+        osc->createLink(loc+string("Penvval") + to_s(i), this);
     }
 }
 
-- 
2.47.0

